cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/License.hs

```diff
@@ -40,128 +41,33 @@ fromCabalLicense AllRightsReserved                         = Known "lib.licenses
 fromCabalLicense (Apache Nothing)                          = Known "lib.licenses.asl20"
 fromCabalLicense (Apache (Just (versionNumbers -> [2,0]))) = Known "lib.licenses.asl20"
 fromCabalLicense ISC                                       = Known "lib.licenses.isc"
-fromCabalLicense OtherLicense                              = Unknown Nothing
+fromCabalLicense OtherLicense                              = Known "lib.licenses.unfree"
```

Comment:
This is incorrect. We assume all licenses are not unfree since Hackage requires open source licenses.